### PR TITLE
RDM-1591 Added conditional hide/show for case view

### PR DESCRIPTION
**Pull request by tzarsmango**
https://tools.hmcts.net/jira/browse/RDM-1591

This one is adding conditional hide/show for the CaseView Fields.

The case can be tested using the `stubs-unit` application

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/shared/conditional-show/conditional-show-view.directive.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { AfterViewInit, Directive, ElementRef, Input } from '@angular/core';
+import { CaseField } from '../domain/definition/case-field.model';
+import { ShowCondition } from './conditional-show.model';
+
+@Directive({selector: '[ccdConditionalShowView]'})
```
Comment (mario-paniccia) [inline src/app/shared/conditional-show/conditional-show-view.directive.ts]: why a new directive? 
have you tried using  ccdConditionalShow without passing the formGroup property? e.g.
<div ccdConditionalShowView ccdLabelSubstitutor [caseField]="field" [eventFields]="caseFields">

Comment (tzarsmango) [reply src/app/shared/conditional-show/conditional-show-view.directive.ts]: I started with that option but got into trouble -which I don't remember clearly now- so I decided to remove some logic to have a simple new directive

Comment (andrewfolga) [reply src/app/shared/conditional-show/conditional-show-view.directive.ts]: Would be nice to reuse the existing directive that @mario-paniccia suggested if it is not too hard to do?

**Comment by tzarsmango** [comment]
The fix for RDM-1591 will be delivered by `RDM-1964`. Will use the existing `ccdConditionalShow` directive as well. So, closing this PR.

**Pull request closed without merging**